src/envo/envo-edit.owl

```diff
@@ -40526,6 +40530,40 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105019> "quality of a foam"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105019> <http://purl.obolibrary.org/obo/PATO_0001018>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105020> (ingestion)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Ingestion>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105020> "A feeding behavior during which material is taken into the body of an organism."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105020> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105020> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105020> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105020> "2021-05-11T14:09:27Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105020> "(Nahrungs)aufnahme"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105020> "Ingestion"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105020> "ingestione"@it)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105020> "ingestión"@es)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105020> "攝食"@zh)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105020> "envoPlastics"@en)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_06105020> "Ingestion of materials is not limited to food, as a pathogen can also transmit through ingestion.
+
+https://github.com/EnvironmentOntology/envo/pull/1153"@en)
```

Comment:
This is a little off - a pathogen can also be food. Particles of indigestible material is a better example. We should also note that ingestion is not limited to organisms with a mouth and digestive tract, but also to things like amoebae.